content/webapp/pages/works/[workId]/index.tsx

```diff
@@ -55,9 +67,18 @@ export const getServerSideProps: GetServerSideProps<
 
   const { url, ...work } = workResponse;
 
+  const iiifPresentationLocation = getDigitalLocationOfType(
+    work,
+    'iiif-presentation'
+  );
+  const iiifManifest =
+    iiifPresentationLocation &&
+    (await fetchIIIFPresentationManifest(iiifPresentationLocation.url));
+
   return {
     props: serialiseProps({
       work,
+      hasIIIFManifest: !!iiifManifest,
```

Comment:
So I have a performance concern that we have to wait for the work to be returned, then have to wait for the manifest to be returned.  However, if we are doing this then can't we pass the entire manifest through so we don't have to get it again client side?

Reply:
If we pass the entire manifest through, so we don't have to request it again client side then we should do that for everything I think.